domains/hjgaming.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "discord connected domain",
+  "repo": "https://github.com/Hjgaming/register-1",
+  "owner": {
+    "username": "hjgaming",
+    "email": "jariwalaharshal80@gmail.com"
+  },
+  "record": {
+    "TXT": ["vc-domain-verify=hjgaming.is-a.dev,b1a85aa1f9f16d27c005"]
```

Comment:
Vercel verification records should be under a file called _vercel.domain.json, this file should be the actual A or CNAME records.